cmake/VcpkgToolchain.cmake

```diff
@@ -22,6 +22,12 @@ if(NOT USE_VCPKG)
     return()
 endif()
 
+include(SubmoduleUtils)
+plasma_init_git_submodule("vcpkg" RESULT has_vcpkg_module)
+if(NOT has_vcpkg_module)
+    return()
+endif()
```

Comment:
To make errors a little bit clearer, we might want to move this above the "Welcome to vcpkg-land" bit, and change `option(USE_VCPKG` to use [`cmake_dependent_option()`](https://cmake.org/cmake/help/latest/module/CMakeDependentOption.html) to clear out `USE_VCPKG` as a hint that vcpkg can't be enabled.

Reply:
I had been hoping to avoid cloning the submodule if vcpkg hadn't been turned on

Reply:
Hmm... Maybe we can defer this for another addition to the submodule tools.